packages/design-system/src/components/Table/TableBody.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+export interface TableBodyProps {
+  /**
+   * The table body contents, usually `TableRow`.
```

Comment:
Can a `tbody` legally have any direct descendants other than a `tr`? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tbody

Reply:
Yes, the MDN states that it should be `tr` only. I find that if a html page with `tbody` that contains children other then `tr`, console errors/warning are usually shown